logan/docker-images/v1.0/oraclelinux/8/entrypoint.sh

```diff
@@ -1,6 +1,3 @@
 #!/usr/bin/env sh
 
-# Copyright (c) 2023, Oracle and/or its affiliates.
```

Comment:
Oracle License small text is required in all the files.

Reply:
Done